web/test-utils/src/jsMain/kotlin/org/jetbrains/compose/web/testutils/TestUtils.kt

```diff
@@ -21,13 +21,18 @@ import kotlin.time.DurationUnit
 import kotlin.time.ExperimentalTime
 import kotlin.time.toDuration
 
+interface TestComposition {
```

Comment:
didn't you plan to remove this interface?

Reply:
I have no idea how this end up here - I apologise for this inconvenience and will rebase this PR on new master